qdrant-landing/content/documentation/frameworks/langchain4j.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Langchain4J
+weight: 110
+---
+
+# Langchain4J
+
+[Langchain4J](https://github.com/langchain4j/langchain4j) is a community port of [Langchain](https://www.langchain.com/) for building context-aware AI applications in Java
```

Comment:
Based on our discussion of maven... I didn't remember, "4J" refers to Java! So I'm thinking:

```suggestion
# LangChain for Java

LangChain for Java, also known as [Langchain4J](https://github.com/langchain4j/langchain4j), is a community port of [Langchain](https://www.langchain.com/) for building context-aware AI applications in Java
```